hail/python/hail/docs/cloud/query_on_batch.rst

```diff
@@ -54,7 +54,8 @@ Getting Started
 
     hailctl config set query/backend batch
 
-6. Now you are ready to `try Hail <../install/try.rst>`__!
+6. Now you are ready to `try Hail <../install/try.rst>`__! If you want too switch back to Spark
```

Comment:
- `too` -> `to`
- delete 'powered Query-on-Batch` or make it `from powered ... `
-